### PR TITLE
Add resume selection documentation for student hiring

### PR DESCRIPTION
**Pull request by carolyncole**

### CODE REVIEW

File: student-software-developers/administration-handbook/resume-selection.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * Resume Reviwer
+
+### Resume Acceptor
+The Resume Acceptor is the person that gathers resumes from the students.  
+The resume acceptor: 
+   1. Has thier email on the job posting
+   1. Reaches out to the student if the information is not complete.
+   1. Redacts Personal Names and Buisness Names from the Resume and cover letters
+   1. Stores the information under the student netid on a google drive accessible to the committee
+   1. Creates and maintains a spreadheet that has the netid, a link to the content and a slot for two reviewers
```
Comment (carolyncole) [inline student-software-developers/administration-handbook/resume-selection.md:17]:
```suggestion
   1. Creates and maintains a spreadsheet that has the netid, a link to the content and a slot for two reviewers  [see template](https://docs.google.com/spreadsheets/d/1rw_kw7psuiWlvJtjbfwyXjWxPsyfNCqSIWafj4Gm5YE/edit#gid=0) 
```

**Review by hackartisan — CHANGES_REQUESTED** [review]
This looks great and I think it reflects what we discussed. I wonder how it should interact with the info on https://github.com/pulibrary/pul-it-handbook/blob/main/student-software-developers/administration-handbook/roles.md. Maybe the roles should be moved there and we add a reference from this page? 

I also think it would be good to add links to this page from the January and February pages, since it's relevant to some of the stuff on those.

**Comment by carolyncole** [comment]
@hackartisan since that role page is just a list it did not make much sense to change it, but I did link those role to the information in the more detailed page about resume selection.

Can you give me more information on what you want in the January February pages?

**Review by hackartisan — APPROVED** [review]
I feel good about this! I think it would be nice for @christinach to look over and ensure it is clear; I'm going to add a reviewer request.

**Review by christinach — APPROVED** [review]
Thank you it looks great!

**Pull request merged**